ostelco-ios-client.xcodeproj/project.pbxproj

```diff
@@ -1797,6 +1802,7 @@
 				042928F9223D4D8800806438 /* SignUpCompletedViewController.swift in Sources */,
 				04834D732237B11800A01500 /* CustomerModel.swift in Sources */,
 				C29802AD2181EDA800FBC5C3 /* OstelcoAPI.swift in Sources */,
+				04D6C9862276E32700AE13DD /* Analytics.swift in Sources */,
```

Comment:
If you're gonna prefix a class, please also prefix the filename (makes it WAY easier to find)

Reply:
Of course!